benchmarking/generate_l1_svm_lp.jl

```diff
@@ -82,25 +101,82 @@ function parse_command_line()
   return ArgParse.parse_args(arg_parse)
 end
 
+"""
+Loads a LIBSVM file and returns a loaded SvmTrainigData struct.
+"""
+function load_libsvm_file(file_name::String)
+  open(file_name, "r") do io
+    labels = Vector{Float64}()
+    row_indices = Vector{Int64}()
+    col_indices = Vector{Int64}()
+    matrix_values = Vector{Float64}()
+
+    row_index = 0
+    for line in eachline(io)
+      row_index += 1
+      split_line = split(line)
+
+      label = parse(Float64, split_line[1])
+      # This ensures that labels are 1 or -1. Different datasets use {-1, 1}, {0, 1}, and {1, 2}.
+      if label ≈ 1.0
+        label = 1.0
+      else
+        label = -1.0
+      end
+      push!(labels, label)
+
+      for i in 2:length(split_line)
+        push!(row_indices, row_index)
+        matrix_coef = split(split_line[i], ":")
+        push!(col_indices, parse(Int64, matrix_coef[1]))
+        push!(matrix_values, parse(Float64, matrix_coef[2]))
+      end
+    end
+    feature_matrix = sparse(row_indices, col_indices, matrix_values)
+    return SvmTrainigData(feature_matrix, labels)
+  end
+end
+
+function normalize_columns(feature_matrix::SparseMatrixCSC{Float64,Int64})
+  norm_of_columns = vec(sqrt.(sum(t -> t^2, feature_matrix, dims = 1)))
+  norm_of_columns[iszero.(norm_of_columns)] .= 1.0
+  return feature_matrix * Diagonal(1.0 ./ norm_of_columns)
+end
+
+function remove_empty_columns(feature_matrix::SparseMatrixCSC{Float64,Int64})
+  keep_cols = Array{Int64,1}()
```

Comment:
nit: one more place where you could use Vector{Int64} instead of Array{Int64,1}

Reply:
Done.